src/handlers/command_handler.rs

```diff
@@ -25,8 +25,9 @@ pub fn process(manager: &mut Manager, command: Command, val: Option<String>) ->
         Command::GotoTag if val.is_none() => false,
         Command::GotoTag if !is_num(&val) => false,
         Command::GotoTag => {
-            let current_tag = manager.tag_index(manager.focused_tag(0).unwrap_or("".to_string()));
-            let previous_tag = manager.tag_index(manager.focused_tag(1).unwrap_or("".to_string()));
+            let current_tag = manager.tag_index(manager.focused_tag(0).unwrap_or_else(String::new));
```

Comment:
Consider `unwrap_or_default`.

Reply:
Ah, that's what it was. Was trying to remember what the right way to do it was, since Clippy suggested .unwrap_or_else(|| String::from("")) which is subpar. 

Thanks for pointing that out!